res/mixxx.qrc

```diff
@@ -29,6 +29,7 @@
         <file>images/library/ic_library_traktor.svg</file>
         <file>images/library/ic_library_rekordbox.svg</file>
         <file>images/library/ic_library_serato.svg</file>
+        <file>images/library/ic_library_tracksuggestion.svg</file>
```

Comment:
@rinso0 
Do you have intest to help with the suggestion icon? 
I have something in mind like a lighting bulb or a speech bubble.

Reply:
A light bulb is perfect!
I'll see if I can come up with some clean SVG icon

Reply:
Great, Thank you :-)